to_sandbox.txt

```diff
@@ -1,12 +1,47 @@
 # New
+ofl/briemhand # https://github.com/google/fonts/pull/7040
+ofl/edunswacthand # https://github.com/google/fonts/pull/7403
+ofl/eduqldhand # https://github.com/google/fonts/pull/7404
+ofl/edusahand # https://github.com/google/fonts/pull/7405
+ofl/eduvicwanthand # https://github.com/google/fonts/pull/7402
+ofl/jaro # https://github.com/google/fonts/pull/7391
+ofl/teachers # https://github.com/google/fonts/pull/7392
 
 # Upgrade
+ufl/ubuntusans # https://github.com/google/fonts/pull/7353
+ufl/ubuntusansmono # https://github.com/google/fonts/pull/7353
 
 # Designer profile
+catalog/designers/afrotype # https://github.com/google/fonts/pull/7499
+catalog/designers/andreasrasmussen # https://github.com/google/fonts/pull/7379
+catalog/designers/celinehurka # https://github.com/google/fonts/pull/7477
+catalog/designers/chankdiesel # https://github.com/google/fonts/pull/7462
+catalog/designers/gunnlaugurbriem # https://github.com/google/fonts/pull/7475
+catalog/designers/orangered # https://github.com/google/fonts/pull/7337
 
 # Metadata / Description / License
+ofl/danfo # https://github.com/google/fonts/pull/7498
+ofl/jacquard12 # https://github.com/google/fonts/pull/7302
+ofl/joan # https://github.com/google/fonts/pull/7351
+ofl/kaushanscript # https://github.com/google/fonts/pull/7377
+ofl/lekton # https://github.com/google/fonts/pull/7356
+ofl/loveyalikeasister # https://github.com/google/fonts/pull/7346
+ofl/notosansmongolian # https://github.com/google/fonts/pull/7472
+ofl/notosanssymbols # https://github.com/google/fonts/pull/7480
+ofl/ojuju # https://github.com/google/fonts/pull/7333
```

Comment:
The Ojuju metadata update is in sandbox.  Let's remove https://github.com/google/fonts/pull/7333 from to_sandbox and add it to to_production. Thanks!